neps/nep-519-yield-execution.md

```diff
@@ -0,0 +1,192 @@
+---
+NEP: 519
+Title: Yield Execution
+Authors: Akhi Singhania <akhi3030@gmail.com>
+Status: Draft
+DiscussionsTo: https://github.com/near/NEPs/pull/519
+Type: Protocol
+Version: 0.0.0
+Created: 2023-11-17
+LastUpdated: 2023-11-20
+---
+
+## Summary
+
+Today, when a smart contract is called by a user or another contract, it has no sensible way to delay responding to the caller.  There exist use cases where contracts would benefit from being able to delay responding till some arbitrary time in the future.  This proposal introduces such possibility into the NEAR protocol.
```

Comment:
> till some arbitrary time

This feels potentially misleading to me given that we enforce a timeout. Could it be clearer to say something like "await a future transaction" rather than "delay responding"?

Reply:
Hopefully improved now.